scraper.py

```diff
@@ -0,0 +1,169 @@
+import selenium
+import re
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.common.by import By
+from selenium.webdriver.chrome.service import Service
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from webdriver_manager.chrome import ChromeDriverManager
+
+driver = webdriver.Chrome(service=Service(ChromeDriverManager().install()))
+# PATH = "/Users/fpw/Desktop/CDP/chromedriver"
+# driver = webdriver.Chrome(PATH)
+
+driver.get("https://atlantacityga.iqm2.com/Citizens/SplitView.aspx?Mode=Video&MeetingID=3587&Format=Minutes")
+#print(driver.title)
+#print(driver.page_source)
```

Comment:
There are two types of things here.

1. If you don't think you will ever need these print statements again, just get rid of them. You can always revert back a commit if something breaks in the future. Many small commits is better than few large commits.
2. If you think you will want these in the future, we should convert them to logging. I can teach.